docs/eclipse/release-notes.md

```diff
@@ -5,18 +5,35 @@ title: Release Notes
 
 Here you will find the latest information about releases of the jSparrow Eclipse plugin.
 
+## 3.2.0
+
+_21.02.2019_
+
+### jSparrow is Fit for Java 11
+
+Since the official End of Life of Java 8 has been reached, we have updated jSparrow to be able to run on Java 11 as well as Java 8.
+
+### New Rule
+
+jSparrow 3.2.0 brings a new rule for [collapsing if statements](/rules/collapse-if-statements.html). 
+The motivation behind this rule is to improve the readability of the code by reducing the number of the nested language constructs. 
```

Comment:
I would remove "the" twice because we are talking about improving readability of code in general: 

The motivation behind this rule is to improve readability of code by reducing the number of nested language constructs.